src/http-auth-interceptor.js

```diff
@@ -5,6 +5,12 @@
  * (c) 2012 Witold Szczerba
  * License: MIT
  */
+
+/* commonjs package manager support (eg componentjs) */
+if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
+  module.exports = 'http-auth-interceptor';
```

Comment:
I am not sure I get it from the link. I mean it still seems incredibly wired to me to export just the name of the module, instead of the module itself.
I do in Node.JS a lot, it has the CommonJS export feature, but I have never seen exporting the name of the module as a "string". What's the possible use if all ask a module system to import a named module and getting the module's name in return...